user/languages/objective-c.md

```diff
@@ -41,8 +41,8 @@ the following table:
 
 ## Default Test Script
 
-Travis CI runs [xctool](https://github.com/facebook/xctool) by default to
-execute your tests. In order for xctool to work, you need to tell it where to
+Travis CI runs xcodebuild and [xcpretty](https://github.com/supermarin/xcpretty) by default to
+execute your tests. In order for xcodebuild to work, you need to tell it where to
 find your project or workspace and what scheme you would like to build. For
```

Comment:
I think underneath this, there's an example field that specifies the minimal config: 

```
language: objective-c
xcode_project: MyNewProject.xcodeproj # path to your xcodeproj folder
xcode_scheme: MyNewProjectTests
```
Would it make sense for us to expand it with xcode_destination too? As to make the example complete?

```
language: objective-c
xcode_project: MyNewProject.xcodeproj # path to your xcodeproj folder
xcode_scheme: MyNewProjectTests
xcode_destination: # example here
```

t'd be great if we use a different example than the one we'll use in our blog :)

Reply:
I think so!